packages/p2p-v2/src/components/AdvertiserName/AdvertiserNameStats.tsx

```diff
@@ -13,16 +16,34 @@ import './AdvertiserNameStats.scss';
  *
  * Use cases are to show this in My Profile and Advertiser page
  */
-const AdvertiserNameStats = () => {
-    const { data: advertiserStats } = useAdvertiserStats();
+const AdvertiserNameStats = ({ advertiserStats }: { advertiserStats: TAdvertiserStats }) => {
     const { isMobile } = useDevice();
+    const isMyProfile = getCurrentRoute() === 'my-profile';
 
-    const { blocked_by_count, daysSinceJoined, rating_average, rating_count, recommended_average } =
-        advertiserStats || {};
+    const {
+        blocked_by_count,
+        daysSinceJoined,
+        is_online,
+        last_online_time,
+        rating_average,
+        rating_count,
+        recommended_average,
```

Comment:
Does this work? so that you don't have to use `/* eslint-disable camelcase */` 🤔 
```suggestion
        blocked_by_count: blockedByCount,
        daysSinceJoined,
        isOnline,
        lastOnlineTime,
        ratingAverage,
        ratingCount,
        recommendedAverage,
```